src/ReactTinyLink.tsx

```diff
@@ -144,8 +144,10 @@ ReactTinyLink.defaultProps = {
   cardSize: 'small',
   maxLine: 2,
   minLine: 1,
+  header: null,
+  descript: null,
```

Comment:
typo 👀 ,
I think the prop name is `description `

Reply:
While this is not an excuse and it is my mistake. This happens when there are not type definitions.
I did rewrite part of the components to use type validations.
Hope you don't mind.